### ISSUE TITLE
Doesn't work with Undertale 1.001 Beta

### ISSUE THREAD

**Issue by NarryG** [lynn/GMSD#3]
Crashes with an exception about negative numbers

**Comment by lynn** [comment]
Yes, the GM:S bytecode format seemed to have changed between versions. This will take a bit of an overhaul to fix... Thanks for the report!

**Comment by NarryG** [comment]
Looks like both the Flowey the Decompiler guy and someone else are also working on reversing the updated Bytecode
https://gist.github.com/PoroCYon/45f947d576f715de3a4d

**Comment by PoroCYon** [comment]
@danne2 FYI, [my implementation](https://github.com/PoroCYon/Altar.NET/) is already functional! >:D

```
0x000000:
    self.demona = "Greetings."
    self.demonb = "You have made yourself completely clear."
    self.demonc = "Understood."
    self.demond = "I, your humble servant, will follow you to the utmost..."
    exit
```

**Comment by VelocityRa** [comment]
@PoroCYon Nice! Thanks for documenting everything as well.
